modules/fuzzy/samples/fuzzy_filtering.cpp

```diff
@@ -0,0 +1,31 @@
+#include "opencv2/core.hpp"
```

Comment:
it would be very nice if you provide a short textual description (in a comment) - what each sample does